src/main/java/Task.java

```diff
@@ -0,0 +1,47 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private static int taskArraySize = 0;
+
+    public Task (String description) {
+        this.description = description;
+        this.isDone = false;
+        taskArraySize++;
+    }
+
+    @Override
+    public void finalize() {
+        taskArraySize--;
+    }
+
+    public String printString() {
+        return "";
+    }

```

Comment:
- Class java.lang.Object has a default toString() method. You should override this method instead of creating a new method. This applies for all instances of printString() in this program.
- Confusing method name: I would expect printString() to print the string using System.out.println(), not return a string type object.